FlowCrypt/App/Configuration.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Configuration.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 05.06.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+class Configuration {
+    static let excludedDomains = ["gmail.com", "googlemail.com", "outlook.com"]
```

Comment:
Let's call it `publicEmailProviderDomains` - more descriptive of the content

Reply:
done